agent/engine/docker_task_engine.go

```diff
@@ -221,7 +221,7 @@ func (engine *DockerTaskEngine) MustInit(ctx context.Context) {
 		err := engine.Init(ctx)
 		if err != nil {
 			errorOnce.Do(func() {
-				log.Error("Could not connect to docker daemon", "err", err)
+				seelog.Errorf("Task engine: could not connect to docker daemon: %v", err)
```

Comment:
nit: should we tag these as `DockerTaskEngine` to keep parity with naming style as we have for `DockerGoClient`?

Reply:
TL;DR I don't think so. Let me know what you think.

I used `DockerGoClient` as I couldn't think of a better name for this. `"Task engine"` distinguishes this from things like `"Stats engine"`. I don't know if we need very strict rules about this.

Reply:
Okay that's fine with me. I don't feel strongly either way. I'm guessing we'll have a better idea after a couple of weeks of log diving and debugging with these new changes 😛